packages/@lwc/template-compiler/src/parser/utils/html-element-attributes.ts

```diff
@@ -31,10 +31,12 @@
  */
 import { hasOwnProperty } from '@lwc/shared';
 
+// This mapping is based on https://github.com/wooorm/html-element-attributes/blob/270d8ce/index.js
```

Comment:
This is where the original list came from, but it was not mentioned anywhere. Probably it was taken from an older version of the library, so I just updated it.

Reply:
Side note: I would love to switch to just using the `html-element-attribute` library directly (as well as `svg-element-attribute`), but they dropped support for CommonJS, so we can't switch until we fix #3017.

Reply:
Opened #3019 to track this.